internal/application/filesystem.go

```diff
@@ -319,7 +319,8 @@ func runPlan(appName string, debugMode bool) bool {
 		for scannerStdout.Scan() {
 			text := scannerStdout.Text()
 			var re = regexp.MustCompile(timestampRegex)
-			updatedText := re.ReplaceAllString(text, m2kString)
+			replacementString := "App name: " + appName + "; " + m2kString
```

Comment:
"Move2Kube" is kind of obvious since the move2kube-api is also for Move2Kube only. Is that right? So I guess we can ignore it.

Reply:
Yes sure, I will remove the "Move2Kube" string. Earlier, I kept the "Move2Kube" string to separate the two- `level=info msg="Move2Kube level=info msg ...`. But, now we also have the App name and artifact name in between them `level=info msg="App name: new123; Artifact name:new123_1610521364; Move2Kube level=info msg` so the "Move2Kube" can be ignored.

Reply:
I have updated the code, now "Move2Kube" is removed.